backend/db_migrations.py

```
@@ -342,7 +358,98 @@ def apply_migration(conn: sqlite3.Connection, from_version: int, to_version: int
+                            pass
+                        else:
+                            raise
+                conn.commit()
+
+                # Seed the curated cancer taxonomy. Import lazily so a missing
+                # seed module does not break older migrations.
+                try:
+                    from cancer_taxonomy_seed import seed_cancer_taxonomy
+                    seeded = seed_cancer_taxonomy(conn)
+                    print(f"   🌱 Seeded {seeded['types']} cancer types, {seeded['subtypes']} subtypes")
+                except Exception as seed_err:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)